repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterRouterImpl.java

```diff
@@ -193,7 +193,8 @@ public void route(HttpServletRequestWrapper servletRequest, HttpServletResponse
                             LOG.error("Error reading request content", e);
                             servletResponse.sendError(HttpServletResponse.SC_REQUEST_ENTITY_TOO_LARGE, "Error reading request content");
                         } else {
-                            LOG.error("Connection Refused to {}", location.getUri(), e);
+                            LOG.error("Error communicating with {}", location.getUri());
+                            LOG.trace("", e);
```

Comment:
Would you be okay with adding the exception message to the error log?

https://github.com/rackerlabs/repose/blob/master/repose-aggregator/components/services/akka-http-client-service/akka-http-client-service-impl/src/main/java/org/openrepose/core/services/serviceclient/akka/impl/AkkaServiceClientImpl.java#L112-L113